scripts/init.zsh

```diff
@@ -58,6 +58,10 @@ zstyle -T :znap: auto-compile &&
     add-zsh-hook preexec ..znap.auto-compile
 add-zsh-hook zsh_directory_name ..znap.dirname
 
+if zstyle -s ':znap:pull' auto-update _ ; then
+    .znap.auto-update
```

Comment:
I don't know whether the function should be `.znap.auto-update` or `..znap.auto-update` -- the intent behind the naming difference in the functions is not clear to me. Happy to adjust if necessary.

Reply:
Oh, I see -- .znap.foo is a `znap` subcommand. I'll rename to `..znap.auto-update`.